pkg/vulnsrc/ubuntu/testdata/vuln-list/ubuntu/CVE-2020-1234.json

```diff
@@ -19,10 +19,11 @@
       "bionic": {
         "Status": "released",
         "Note": "1.2.3"
-      },
-      "focal": {
-        "Status": "needs-triage",
-        "Note": ""
+      }
+    },
+    "trusty": {
```

Comment:
The key must be a package name. Isn't `trusty` a codename?

Reply:
Ah yes it should be. I read `xen` as `xenial`, Will change the test data and get it pushed

Reply:
test data updated and pushed.